.github/workflows/checkstyle.yaml

```diff
@@ -5,23 +5,28 @@ on:
   pull_request_target:
 
 jobs:
-  deploy:
+  checkstyle:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
       with:
         ref: ${{ github.event.pull_request.head.sha }}
     - name: Install dependencies
       run: |
+        sudo apt-get update
         sudo apt-get install --yes -qq build-essential autoconf libtool gawk alien fakeroot linux-headers-$(uname -r)
         sudo apt-get install --yes -qq zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev libssl-dev python-dev python-setuptools python-cffi python3 python3-dev python3-setuptools python3-cffi
         # packages for tests
         sudo apt-get install --yes -qq parted lsscsi ksh attr acl nfs-kernel-server fio
-        sudo apt-get install --yes -qq mandoc flake8 # devscripts - enable then bashisms fixed
+        sudo apt-get install --yes -qq mandoc cppcheck pax-utils # devscripts - enable then bashisms fixed
```

Comment:
The bashism warnings should have been resolved by 64025fa3a1f0f710f7f8678f2ac459b07ed9f88f and 72eedb69fd8a15ea2b6ac3ef2fa322210dd42f6c.  It'd be nice to enable this as well.

Reply:
@gmelikov I went ahead and merged this as is.  We can follow up with enabling the bashism checks once we're sure they'll pass.